src/App.tsx

```diff
@@ -38,6 +39,11 @@ interface L1ToL2MessageReaderWithNetwork extends L1ToL2MessageReader {
   l2Network: L2Network;
 }
 
+const isEthDeposit = async (l1ToL2Message:L1ToL2MessageReader ): Promise<boolean>=>{
+  const txData = (await l1ToL2Message.l2Provider.getTransaction(l1ToL2Message.retryableCreationId)).data
+  // check that calldataSize param is zero (8th 32-byte param, offset by 4 bytes for message ID):
```

Comment:
I think this works / am going off of https://github.com/OffchainLabs/arb-os/blob/962ab51cffff5d57daee0f4b031d69a3224d51d4/arb_os/retrybuffer.mini#L305

could use a sanity check tho